src/main/webapp/app/shared/delete-dialog/delete-dialog.component.html

```diff
@@ -6,6 +6,14 @@ <h4 class="modal-title" jhiTranslate="entity.delete.title">Confirm delete operat
     <div class="modal-body">
         <jhi-alert-error></jhi-alert-error>
         <p [jhiTranslate]="deleteQuestion" [translateValues]="{ title: entityTitle }">Are you sure you want to delete?</p>
+        <div class="checkbox" *ngIf="checkboxText">
```

Comment:
Yes, I did also think about that. However, there is no cases like that and I believe it wouldn't be needed. And this will make the logic more complicated, because we need to handle array of texts and connect them somehow to array of values that checkbox returns.

Reply:
I think it is ok to **not** handle this case here. Maybe if we need this, it is ok to build a custom ui for that.